chainer_setup_build.py

```diff
@@ -261,12 +261,26 @@ def get_cython_pkg():
     return pkg_resources.get_distribution('cython')
 
 
+def run_command(cmd):
+    try:
+        subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+    except subprocess.CalledProcessError as e:
+        msg = '''Command %r failed:
+
+  command: %s
+  return code: %d
+  output:
+
+%s''' % (cmd[0], ' '.join(e.cmd), e.returncode, e.output)
```

Comment:
The message of stderr is stored as `CalledProcessError.stderr` attribute (c.f. see [here](https://github.com/python/cpython/blob/b7276602e9cc9295c7eb74b515b4822bc791288d/Lib/subprocess.py#L380))

So how about showing both `output` and `stderr` attribute?